packages/@aws-cdk/aws-cloudfront/test/experimental/edge-function.test.ts

```diff
@@ -102,6 +102,14 @@ describe('stacks', () => {
     });
   });
 
+  test('us-east-1 stack inherits account of parent stack', () => {
```

Comment:
Also maybe a test case so that if parent stack account is `undefined`, the edge stack follows.